62/serfer2/scraper.py

```diff
@@ -0,0 +1,71 @@
+# coding=utf-8 -*-
+
+import requests
+from lxml import html
+
+SPLIT_SYMBOLS = [',', ' y ', ' e ', ' - ']
+
+
+class NameScraper(object):
+    """
+    Scrapping names for a given url and xpath
+    """
+
+    _url: str = None
+    _xpath: str = None
+
+    def __init__(self, config):
+        self._url = config['url']
+        self._xpath = config['xpath']
+
+    def get_names(self) -> list:
+        """
+        get all found names
+        """
+        speakers = self.get_items_from_url()
+        for symbol in SPLIT_SYMBOLS:
+            speakers = self.split_names(speakers, symbol)
+        return self.filter_false_names(speakers)
+
+    def get_items_from_url(self) -> list:
+        """
+        Gets speaker's name list from a web page
+        Params:
```

Comment:
params are not up to date, method has no params beside self, which of course in a class is no real param